### PR TITLE
Side menu

### PR DESCRIPTION
**Pull request by puglisac**

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
when u are ready for review, lets add some screen shots

**Comment by puglisac** [comment]
Screen Shot 2021-03-22 at 11 05 42 AM
Was able to get some styling in. Would love thoughts and suggestions!

**Comment by daemyonc** [comment]
Looks good. 

My thought was to keep font-weight consistent between both menus which I had the top menu at `font-weight: normal`. I think the `#add-job-button` looks good at the font-weight you have it but change the color to see the same color as the border. That way it has more contrast from the background. Also, I think maybe we should go with the same width for the search, add job button, and the side-bar highlight.

**Comment by puglisac** [comment]
Screen Shot 2021-03-22 at 9 08 55 PM
Thanks @daemyonc!

**Comment by daemyonc** [comment]
Beautiful! Yeah, I think that looks better. Thanks @puglisac

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <TopNav />
-<SubNav />
\ No newline at end of file
+<SubNav />
+<SideMenu />
```
Comment (ShaunWMusic) [inline app/templates/application.hbs]: honestly, i think we may want different sidemenu per page, so we can accomplish this one of two ways... before we discuss that though, i believe we need to move this component inside Each page(ex. <SideMenu /> would sit inside jobs/Template.hbs file) 
1.) make this SideMenu component render dynamically by adding props that we pass into it per page
2.) make this SideMenu named specific to each page and only show data specific to its page

File: app/index.html [hunk 2]
```diff
@@ -9,9 +9,11 @@
 
     {{content-for "head"}}
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Lora:wght@700&display=swap" rel="stylesheet">    
+    <link href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,700&family=Roboto+Condensed:wght@700&display=swap" rel="stylesheet">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/jobsite-ui.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+
```
Comment (ShaunWMusic) [inline app/index.html:16]: we could also consider bringing in the Fonts, so we don't have as many external links that app is relying on... just a thought...

File: app/components/side-menu/template.hbs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <input type="text" placeholder="&#xF002;" style="font-family:Arial, FontAwesome">
+    </form>
+
+    <div class="side-menu-filters">
+        <p>FILTER BY</p>
+        <div class="side-menu-item active">
+            <a href="#">All</a>
+        </div>
+        <div class="side-menu-item">
+            <a href="#">Full Time</a>
```
Comment (ShaunWMusic) [inline app/components/side-menu/template.hbs]: The naming here will either need to be dynamic based on the page its being rendered in or we need to rename this from side menu to something more specific to jobs page

Comment (puglisac) [reply app/components/side-menu/template.hbs]: I think making it dynamic would be good. Would use less duplicate code I think. I added a controller to jobs and can pass the menu items down that way. Is that a correct approach and what you're thinking?

Comment (ShaunWMusic) [reply app/components/side-menu/template.hbs]: to pull in data, you will use route.hbs file and `Model()` method...
```
 model() {
    return this.store.findAll('photos');
  }
```
 once you do that, then you can simply pull that data into template.hbs file by doing something similar to 
```<ul>
  {{#each this.photos as |p|}}
    <li>
      <LinkTo @route="photos.edit" @model={{p}}>{{p.title}}</LinkTo>
    </li>
  {{/each}}
</ul>

**Comment by daemyonc** [comment]
2a7ad5e78545940c592084f9dea2b07a5a1a60c3
This was me reverting the previous commit. I think I see what you're doing.

File: app/components/side-menu/template.hbs (hunk 3 above)
Comment (daemyonc) [reply app/components/side-menu/template.hbs]: Wouldn't each page have a side-menu and wouldn't the styling be the same? Sure, different parts of the side menu could be rendered and with different data but seems like the class names in this component would just be for styling.

**Review by daemyonc — APPROVED** [review]

**Review by daemyonc — APPROVED** [review]

**Review by daemyonc — APPROVED** [review]

**Review by daemyonc — APPROVED** [review]

**Pull request merged**